src/client/js/Panels/ObjectBrowser/TreeBrowserControl.js

```diff
@@ -783,6 +794,29 @@ define(['js/logger',
             //ENDOF : HANDLE UPDATE
         };
 
+        checkLibraries = function () {
+            var libraryNames = client.getLibraryNames(),
+                i,
+                checkLibrary = function (name) {
+                    self._libraryChecks[name] = true;
+                    self._libraryManager.check(name,
+                        function (err, notified) {
+                            if (!err && !notified) {
+                                delete self._libraryChecks[name];
+                            }
+                        });
+                };
+
+            self._libraryChecks = self._libraryChecks || {};
```

Comment:
Oh I see that the check is being book-kept here.. However is this state ever reset?